### ISSUE TITLE
Weird behaviour with external_id

### ISSUE THREAD

**Issue by Korpyc** [klaviyo/klaviyo-android-sdk#85]
### Description

```markdown
I noticed, that this SDK for some reasons also push external_id to custom properties.

So, I see it twice at profile page at dashboard.
```

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla Android project
- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/klaviyo/klaviyo-android-sdk/issues) or [pull request](https://github.com/klaviyo/klaviyo-android-sdk/pulls).

### The Klaviyo Android SDK version information

1.1.0

**Comment by Korpyc** [comment]
![2023-06-30 22 18 13](https://github.com/klaviyo/klaviyo-android-sdk/assets/43324192/385a6f25-f9fc-468b-a906-516d98b5fd63)

**Comment by ndurell** [comment]
Interesting. I can forward this on to our profiles team and see what they think.

**Comment by evan-masseau** [comment]
Looks like I mixed up a key in that API's payload. Thanks so much for reporting this, we'll roll a fix into our upcoming release!

**Comment by evan-masseau** [comment]
Released! Update to version `1.1.1`
Thanks again!
- Evan
